rosflight/include/rosflight/mavrosflight/config_manager.h

```diff
@@ -0,0 +1,76 @@
+#include <vector>
+#include <future>
+#include <chrono>
+#include <string>
+#include <utility>
+
+#include "mavlink_listener_interface.h"
+#include "mavlink_comm.h"
+
+namespace mavrosflight
+{
+
+  class ConfigManager : public MavlinkListenerInterface
+  {
+  public:
+    typedef struct
+    {
+      bool successful;
+      bool reboot_required;
+      std::string error_message;
+    } config_response_t;
+    ConfigManager(MavlinkComm *const comm);
+    ~ConfigManager();
+    void handle_mavlink_message(const mavlink_message_t &msg) override;
+    std::tuple<bool, uint8_t> get_configuration(uint8_t device);
+    config_response_t set_configuration(uint8_t device, uint8_t config);
+    void request_config_info();
+
+    std::tuple<bool, uint8_t> get_device_from_str(const std::string &name) const;
+    std::tuple<bool, uint8_t> get_config_from_str(uint8_t device, const std::string &name) const;
+    bool is_valid_device(uint8_t device) const;
+    bool is_valid_config(uint8_t device, uint8_t config) const;
+    std::string get_device_name(uint8_t device) const;
+    std::string get_config_name(uint8_t device, uint8_t config) const;
+
+    static constexpr std::chrono::milliseconds timeout{500};
```

Comment:
If you change this to `static constexpr std::chrono::milliseconds timeout = std::chrono::milliseconds(500);` can you get rid of the line in `config_manager.cpp`? Also, since this is a constant, we should probably call it `TIMEOUT`.

Reply:
I changed it, but the extra line is still required. I think they fix the issue in C++17.